contents/newsletter/small-teams.md

```diff
@@ -0,0 +1,115 @@
+
+Each quarter, every small team at PostHog creates objectives and key results, or OKRs. Our executive team will give feedback on this to keep everyone on track. Beyond that, during each quarter each small team is tasked with maintaining a prioritized roadmap to help them achieve their objectives. Each small team is also tasked with speaking to customers. 
+
+And, as each one is like a startup in its own right, they’ve got to cover everything. That means:
+
+* Monitoring relevant metrics, including those covering usage, quality, and revenue
+* Triage and fix bugs related to the products or areas they’re responsible for
+* Assist the support hero in answering related questions
+* Collaborate with other Small Teams such as marketing
+* Become power users of their area of PostHog and use PostHog in their processes
+
+### 3. They have one leader
+
+One of the purposes of the small team structure is to keep a startup’s structure flat even as it scales. That means [minimal layers of management](https://posthog.com/handbook/company/management) and lots of autonomy. 
+
+Each small team has a team lead who is responsible for its performance. It’s not a given that the lead is the most experienced person on the team, but rather the person best-suited to leading the product the team is working on. 
+
+At PostHog, we’ve got an engineering-led culture. And that means that our small teams are normally led by the person with overall responsibility for that product. And that person is very likely to be an engineer.
+
+Team leaders do not automatically equal managers. Team leads are responsible for making sure that teams perform well and for giving direct feedback. Managers have a different remit and are focused on the happiness of direct reports and setting the right context for people to do their jobs, onboarding new hires, and discussing performance issues with the executive team. A team lead’s remit is deliberately more product-focused.
+
+### 4. They have their own mission
+
+Regardless of size or scope, each team has its own mission that feeds into our overall company mission of equipping every developer to build successful products.
+
+On the [marketing team](https://posthog.com/teams/marketing), the mission is to make PostHog a place for product engineers to learn and collaborate. The [infrastructure team’s mission](https://posthog.com/teams/infrastructure) is to make deploying, scaling, and managing PostHog easy, fast, and reliable. But a charter is more than a mission. Small team charters also need to include:
+
+* Long-term goals
+* A description of what the team does
+* A target customer
+* Details of who’s on the team
+* Key metrics
+
+All of this information is in our [handbook](https://posthog.com/handbook) and updated when changes are made and confirmed each quarter. This keeps everyone on the same page and makes it easy for anyone within the company to quickly find out what other teams are working on and why.
```

Comment:
Couple of notes here:

- You introduce the idea of a charter vs mission, but it's not super clear why / what the difference is.
- Can we switch out the marketing team example for another team? I think we need to update our mission as it feels a bit out of date.